packages/ckeditor5-link/src/linkui.js

```diff
@@ -21,8 +21,9 @@ import LinkActionsView from './ui/linkactionsview';
 
 import linkIcon from '../theme/icons/link.svg';
 
+// The regex checks for the protocol syntax ('xxxx://' or 'xxxx:')
+// or non-word characters at the beginning of the link ('/', '#' etc.).
 const protocolRegExp = /^((\w+:(\/{2,})?)|(\W))/i;
-const emailRegExp = /[\w-]+@[\w-]+\.+[\w-]+/i;
 const VISUAL_SELECTION_MARKER_NAME = 'link-ui';
```

Comment:
I can see that we don't have this in our code style, unfortunately.

The rule is to sue `CAPS_LOCK` for such constants. So it should be `PROTOCOL_REG_EXP`.

However, this is duplicated also in the other place :point\_up: .